src/celldega/pre/__init__.py

```diff
@@ -301,11 +302,82 @@ def create_image_tiles(
     """
 
     print("\n========Generating image tiles========")
-    if technology != "Xenium":
-        raise ValueError(
-            f"Unsupported technology: {technology}. Currently, only 'Xenium' is supported."
+    if technology == "Xenium":
+        print('------ xenium')
+        create_image_tiles_xenium(
+            data_dir, path_landscape_files, image_tile_layer=image_tile_layer
+        )
+    elif technology == 'h&e':
+        print('------ h&e')
+
+        image_shape = create_image_tiles_h_and_e(
+            data_dir, path_landscape_files, image_tile_layer=image_tile_layer
+        )
+
+        # raise ValueError(
+        #     f"Unsupported technology: {technology}. Currently, only 'Xenium' is supported."
+        # )
+
+    print("Image tiles created successfully.")
+
+def create_image_tiles_h_and_e(
+    data_dir, path_landscape_files, image_tile_layer
+):
+
+    """
+    Creates image tiles for visualization from the H&E image.
+
+    Args:
+        data_dir (str): Path to the directory containing the data (e.g., morphology_focus_0000.ome.tif).
+        path_landscape_files (str): Path to the directory where the image tiles and pyramid will be saved.
+        image_tile_layer (str, optional): Specifies which image layers to process. Options are 'dapi' (default) or 'all'.
```

Comment:
We probably don't need this arg. I assume you copied from the original function below.